src/types/layers.ts

```diff
@@ -24,9 +25,10 @@ const layerOrder: Record<Layer, number> = {
   [Layer.sapphire]: 2,
   [Layer.emerald]: 3,
   [Layer.cipher]: 4,
+  [Layer.pontusx]: 5,
 }
 
-const hiddenLayers: Layer[] = []
+const hiddenLayers: Layer[] = [Layer.pontusx]
```

Comment:
What we want here is that the layer _is_ enabled in the sense that exploring it fully works, but we don't advertise it in the opening graph or the layer selector menu.

This is a different use-case compared to only enabling the layer in dev builds, but not in production.

Reply:
Are you sure about that? The initial requirement was "we'd support it on the landing page of our explorer".

Reply:
I have opened a discussion about this in the relevant chat.

Reply:
The concept of hidden (but accessible) layers has been confirmed by @kostko on chat, so I think we can close this thread.

Reply:
Just to confirm:
- we don't need hidden layers in Graph because everything is hardcoded there and long term we will show this ParaTime in graph.
- the only place where you want to hide it is Network picker (temporary until Pontus is released). Why we can't just add a condition in layer picker?

Reply:
Besides the layer picker, we also want to hide this layer in the search results.
That is the two locations where this flag is tested.

Instead of adding two independent guards (one for the search, one for the layer selector), I have defined in in `layers.ts`. How would you do it differently?

Reply:
Also, search should be disabled for it. So I guess common configuration for hidden layer, makes sense

Reply:
jinx, you owe me a soda 😂

Reply:
> jinx, you owe me a soda 😂

👍🏻  Please remind me next time we meet.

Meanwhile, let's merge this and continue with other stuff.